conf.pri.mingw

```diff
@@ -0,0 +1,58 @@
+# Adapt these paths
+
+# Point this to the boost include folder
+INCLUDEPATH += $$quote(C:/qBittorrent/boost_1_51_0)
```

Comment:
This and the next files encourage to install each library into its own directory. I have no idea what are the advantages of this method, but it is certainly more complicated than installing everything into a single directory. Why should we "advertise" the complicated approach? What are the advantages?

Reply:
Don't try to turn Windows into Unix :)
On Windows everything usually installs into separated directory.